docs/UserGuide.md

```diff
@@ -102,6 +102,26 @@ Examples:
 * `edit p/91234567 e/johndoe@example.com r/1000`
 * `edit e/peterparker@example.com r/420`
 
+### Increasing a Customer’s Reward points: `incr`
+
+Increases the Reward points of existing Customer in bobaBot.
+
+Format: `incr INCREMENT_VALUE p/PHONE_NUMBER OR incr INCREMENT_VALUE e/EMAIL`
```

Comment:
Similarly as mentioned above, maybe we can put a quick note or an alert emoji to let the users know the change in the convention of the command here:)